// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/UploadFilePickerTab/UploadFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./UploadFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_25b91982',
    tabsContainer: 'tabsContainer_25b91982',
    tabContainer: 'tabContainer_25b91982',
    nav: 'nav_25b91982',
    tabHeaderContainer: 'tabHeaderContainer_25b91982',
    breadcrumbNav: 'breadcrumbNav_25b91982',
    breadcrumbNavItem: 'breadcrumbNavItem_25b91982',
    tab: 'tab_25b91982',
    tabFiles: 'tabFiles_25b91982',
    scrollablePaneWrapper: 'scrollablePaneWrapper_25b91982',
    actionButtonsContainer: 'actionButtonsContainer_25b91982',
    actionButtons: 'actionButtons_25b91982',
    actionButton: 'actionButton_25b91982',
    tabHeader: 'tabHeader_25b91982',
    selectedTab: 'selectedTab_25b91982',
    header: 'header_25b91982',
    xlg: 'xlg_25b91982',
    xxlg: 'xxlg_25b91982',
    xxxlg: 'xxxlg_25b91982',
    xxxxlg: 'xxxxlg_25b91982',
    itemPickerTopBar: 'itemPickerTopBar_25b91982',
    commandBarNoChevron: 'commandBarNoChevron_25b91982',
    fileItem: 'fileItem_25b91982',
    folderItem: 'folderItem_25b91982',
    fileTypeIcon: 'fileTypeIcon_25b91982',
    fileTypeIconIcon: 'fileTypeIconIcon_25b91982',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_25b91982',
    iconColumnHeader: 'iconColumnHeader_25b91982',
    fileRow: 'fileRow_25b91982',
    folderRow: 'folderRow_25b91982',
    emptyFolder: 'emptyFolder_25b91982',
    emptyFolderImage: 'emptyFolderImage_25b91982',
    emptyFolderImageTag: 'emptyFolderImageTag_25b91982',
    emptyFolderTitle: 'emptyFolderTitle_25b91982',
    emptyFolderSubText: 'emptyFolderSubText_25b91982',
    emptyFolderMobile: 'emptyFolderMobile_25b91982',
    emptyFolderPc: 'emptyFolderPc_25b91982',
    emptyFolderIcon: 'emptyFolderIcon_25b91982',
    localTabSinglePreview: 'localTabSinglePreview_25b91982',
    localTabSinglePreviewImage: 'localTabSinglePreviewImage_25b91982',
    localTabFilename: 'localTabFilename_25b91982',
    localTabLabel: 'localTabLabel_25b91982',
    localTabInput: 'localTabInput_25b91982',
};
exports.default = styles;
/* tslint:enable */
