// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/WebSearchTab/WebSearchTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./WebSearchTab.module.css');
var styles = {
    filePicker: 'filePicker_a197fbf8',
    tabsContainer: 'tabsContainer_a197fbf8',
    tabContainer: 'tabContainer_a197fbf8',
    nav: 'nav_a197fbf8',
    tabHeaderContainer: 'tabHeaderContainer_a197fbf8',
    breadcrumbNav: 'breadcrumbNav_a197fbf8',
    breadcrumbNavItem: 'breadcrumbNavItem_a197fbf8',
    tab: 'tab_a197fbf8',
    tabFiles: 'tabFiles_a197fbf8',
    scrollablePaneWrapper: 'scrollablePaneWrapper_a197fbf8',
    actionButtonsContainer: 'actionButtonsContainer_a197fbf8',
    actionButtons: 'actionButtons_a197fbf8',
    actionButton: 'actionButton_a197fbf8',
    tabHeader: 'tabHeader_a197fbf8',
    selectedTab: 'selectedTab_a197fbf8',
    header: 'header_a197fbf8',
    xlg: 'xlg_a197fbf8',
    xxlg: 'xxlg_a197fbf8',
    xxxlg: 'xxxlg_a197fbf8',
    xxxxlg: 'xxxxlg_a197fbf8',
    itemPickerTopBar: 'itemPickerTopBar_a197fbf8',
    commandBarNoChevron: 'commandBarNoChevron_a197fbf8',
    fileItem: 'fileItem_a197fbf8',
    folderItem: 'folderItem_a197fbf8',
    fileTypeIcon: 'fileTypeIcon_a197fbf8',
    fileTypeIconIcon: 'fileTypeIconIcon_a197fbf8',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_a197fbf8',
    iconColumnHeader: 'iconColumnHeader_a197fbf8',
    fileRow: 'fileRow_a197fbf8',
    folderRow: 'folderRow_a197fbf8',
    emptyFolder: 'emptyFolder_a197fbf8',
    emptyFolderImage: 'emptyFolderImage_a197fbf8',
    emptyFolderImageTag: 'emptyFolderImageTag_a197fbf8',
    emptyFolderTitle: 'emptyFolderTitle_a197fbf8',
    emptyFolderSubText: 'emptyFolderSubText_a197fbf8',
    emptyFolderMobile: 'emptyFolderMobile_a197fbf8',
    emptyFolderPc: 'emptyFolderPc_a197fbf8',
    emptyFolderIcon: 'emptyFolderIcon_a197fbf8',
    searchBoxContainer: 'searchBoxContainer_a197fbf8',
    searchBoxMedium: 'searchBoxMedium_a197fbf8',
    searchBox: 'searchBox_a197fbf8',
    dropdownContainer: 'dropdownContainer_a197fbf8',
    filterDropdown: 'filterDropdown_a197fbf8',
    filePickerFolderCardGrid: 'filePickerFolderCardGrid_a197fbf8',
    filePickerFolderCardTile: 'filePickerFolderCardTile_a197fbf8',
    filePickerFolderCardPadder: 'filePickerFolderCardPadder_a197fbf8',
    filePickerFolderCardSizer: 'filePickerFolderCardSizer_a197fbf8',
    filePickerFolderCardImage: 'filePickerFolderCardImage_a197fbf8',
    filePickerFolderCardLabel: 'filePickerFolderCardLabel_a197fbf8',
    bingGridListCell: 'bingGridListCell_a197fbf8',
    bingTile: 'bingTile_a197fbf8',
    bingTileContent: 'bingTileContent_a197fbf8',
    bingTileFrame: 'bingTileFrame_a197fbf8',
    bingTileCheckCircle: 'bingTileCheckCircle_a197fbf8',
    bingTileNamePlate: 'bingTileNamePlate_a197fbf8',
    isSelected: 'isSelected_a197fbf8',
    copyrightLabel: 'copyrightLabel_a197fbf8',
    noResultLabel: 'noResultLabel_a197fbf8',
    bingTileThumbnail: 'bingTileThumbnail_a197fbf8',
    bingGrildList: 'bingGrildList_a197fbf8',
};
exports.default = styles;
/* tslint:enable */
