// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/controls/FolderTile/FolderTile.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FolderTile.module.css');
var styles = {
    filePicker: 'filePicker_30b35d02',
    tabsContainer: 'tabsContainer_30b35d02',
    tabContainer: 'tabContainer_30b35d02',
    nav: 'nav_30b35d02',
    tabHeaderContainer: 'tabHeaderContainer_30b35d02',
    breadcrumbNav: 'breadcrumbNav_30b35d02',
    breadcrumbNavItem: 'breadcrumbNavItem_30b35d02',
    tab: 'tab_30b35d02',
    tabFiles: 'tabFiles_30b35d02',
    scrollablePaneWrapper: 'scrollablePaneWrapper_30b35d02',
    actionButtonsContainer: 'actionButtonsContainer_30b35d02',
    actionButtons: 'actionButtons_30b35d02',
    actionButton: 'actionButton_30b35d02',
    tabHeader: 'tabHeader_30b35d02',
    selectedTab: 'selectedTab_30b35d02',
    header: 'header_30b35d02',
    xlg: 'xlg_30b35d02',
    xxlg: 'xxlg_30b35d02',
    xxxlg: 'xxxlg_30b35d02',
    xxxxlg: 'xxxxlg_30b35d02',
    itemPickerTopBar: 'itemPickerTopBar_30b35d02',
    commandBarNoChevron: 'commandBarNoChevron_30b35d02',
    fileItem: 'fileItem_30b35d02',
    folderItem: 'folderItem_30b35d02',
    fileTypeIcon: 'fileTypeIcon_30b35d02',
    fileTypeIconIcon: 'fileTypeIconIcon_30b35d02',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_30b35d02',
    iconColumnHeader: 'iconColumnHeader_30b35d02',
    fileRow: 'fileRow_30b35d02',
    folderRow: 'folderRow_30b35d02',
    emptyFolder: 'emptyFolder_30b35d02',
    emptyFolderImage: 'emptyFolderImage_30b35d02',
    emptyFolderImageTag: 'emptyFolderImageTag_30b35d02',
    emptyFolderTitle: 'emptyFolderTitle_30b35d02',
    emptyFolderSubText: 'emptyFolderSubText_30b35d02',
    emptyFolderMobile: 'emptyFolderMobile_30b35d02',
    emptyFolderPc: 'emptyFolderPc_30b35d02',
    emptyFolderIcon: 'emptyFolderIcon_30b35d02',
    aboveNameplate: 'aboveNameplate_30b35d02',
    tileContent: 'tileContent_30b35d02',
    foreground: 'foreground_30b35d02',
    grid: 'grid_30b35d02',
    cellContent: 'cellContent_30b35d02',
    content: 'content_30b35d02',
    tile: 'tile_30b35d02',
    invokable: 'invokable_30b35d02',
    namePlate: 'namePlate_30b35d02',
    nameplate: 'nameplate_30b35d02',
    name: 'name_30b35d02',
    isLarge: 'isLarge_30b35d02',
    activity: 'activity_30b35d02',
    signalField: 'signalField_30b35d02',
    compact: 'compact_30b35d02',
    signalFieldValue: 'signalFieldValue_30b35d02',
    folderList: 'folderList_30b35d02',
    link: 'link_30b35d02',
    cell: 'cell_30b35d02',
    listCell: 'listCell_30b35d02',
    label: 'label_30b35d02',
    check: 'check_30b35d02',
    selectable: 'selectable_30b35d02',
    showCheck: 'showCheck_30b35d02',
    selected: 'selected_30b35d02',
    dialogMainOverride: 'dialogMainOverride_30b35d02',
    itemTileBlankCover: 'itemTileBlankCover_30b35d02',
    metadata: 'metadata_30b35d02',
    isSmall: 'isSmall_30b35d02',
    listPage: 'listPage_30b35d02',
    odItemTile2FolderCover: 'odItemTile2FolderCover_30b35d02',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_30b35d02',
    folderCover: 'folderCover_30b35d02',
    isSmall_88d29bd4: 'isSmall_88d29bd4_30b35d02',
    folderCoverBack: 'folderCoverBack_30b35d02',
    folderCoverFront: 'folderCoverFront_30b35d02',
    folderCoverFrame: 'folderCoverFrame_30b35d02',
    folderCoverContent: 'folderCoverContent_30b35d02',
};
exports.default = styles;
/* tslint:enable */
