// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/LinkFilePickerTab/LinkFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./LinkFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_e5da0d73',
    tabsContainer: 'tabsContainer_e5da0d73',
    tabContainer: 'tabContainer_e5da0d73',
    nav: 'nav_e5da0d73',
    tabHeaderContainer: 'tabHeaderContainer_e5da0d73',
    breadcrumbNav: 'breadcrumbNav_e5da0d73',
    breadcrumbNavItem: 'breadcrumbNavItem_e5da0d73',
    tab: 'tab_e5da0d73',
    tabFiles: 'tabFiles_e5da0d73',
    scrollablePaneWrapper: 'scrollablePaneWrapper_e5da0d73',
    actionButtonsContainer: 'actionButtonsContainer_e5da0d73',
    actionButtons: 'actionButtons_e5da0d73',
    actionButton: 'actionButton_e5da0d73',
    tabHeader: 'tabHeader_e5da0d73',
    selectedTab: 'selectedTab_e5da0d73',
    header: 'header_e5da0d73',
    xlg: 'xlg_e5da0d73',
    xxlg: 'xxlg_e5da0d73',
    xxxlg: 'xxxlg_e5da0d73',
    xxxxlg: 'xxxxlg_e5da0d73',
    itemPickerTopBar: 'itemPickerTopBar_e5da0d73',
    commandBarNoChevron: 'commandBarNoChevron_e5da0d73',
    fileItem: 'fileItem_e5da0d73',
    folderItem: 'folderItem_e5da0d73',
    fileTypeIcon: 'fileTypeIcon_e5da0d73',
    fileTypeIconIcon: 'fileTypeIconIcon_e5da0d73',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_e5da0d73',
    iconColumnHeader: 'iconColumnHeader_e5da0d73',
    fileRow: 'fileRow_e5da0d73',
    folderRow: 'folderRow_e5da0d73',
    emptyFolder: 'emptyFolder_e5da0d73',
    emptyFolderImage: 'emptyFolderImage_e5da0d73',
    emptyFolderImageTag: 'emptyFolderImageTag_e5da0d73',
    emptyFolderTitle: 'emptyFolderTitle_e5da0d73',
    emptyFolderSubText: 'emptyFolderSubText_e5da0d73',
    emptyFolderMobile: 'emptyFolderMobile_e5da0d73',
    emptyFolderPc: 'emptyFolderPc_e5da0d73',
    emptyFolderIcon: 'emptyFolderIcon_e5da0d73',
    linkTextField: 'linkTextField_e5da0d73',
    localTabSinglePreview: 'localTabSinglePreview_e5da0d73',
    localTabSinglePreviewImage: 'localTabSinglePreviewImage_e5da0d73',
};
exports.default = styles;
/* tslint:enable */
