// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/OneDriveFilesTab/OneDriveFilesTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./OneDriveFilesTab.module.css');
var styles = {
    filePicker: 'filePicker_09fc3e06',
    tabsContainer: 'tabsContainer_09fc3e06',
    tabContainer: 'tabContainer_09fc3e06',
    nav: 'nav_09fc3e06',
    tabHeaderContainer: 'tabHeaderContainer_09fc3e06',
    breadcrumbNav: 'breadcrumbNav_09fc3e06',
    breadcrumbNavItem: 'breadcrumbNavItem_09fc3e06',
    tab: 'tab_09fc3e06',
    tabFiles: 'tabFiles_09fc3e06',
    scrollablePaneWrapper: 'scrollablePaneWrapper_09fc3e06',
    actionButtonsContainer: 'actionButtonsContainer_09fc3e06',
    actionButtons: 'actionButtons_09fc3e06',
    actionButton: 'actionButton_09fc3e06',
    tabHeader: 'tabHeader_09fc3e06',
    selectedTab: 'selectedTab_09fc3e06',
    header: 'header_09fc3e06',
    xlg: 'xlg_09fc3e06',
    xxlg: 'xxlg_09fc3e06',
    xxxlg: 'xxxlg_09fc3e06',
    xxxxlg: 'xxxxlg_09fc3e06',
    itemPickerTopBar: 'itemPickerTopBar_09fc3e06',
    commandBarNoChevron: 'commandBarNoChevron_09fc3e06',
    fileItem: 'fileItem_09fc3e06',
    folderItem: 'folderItem_09fc3e06',
    fileTypeIcon: 'fileTypeIcon_09fc3e06',
    fileTypeIconIcon: 'fileTypeIconIcon_09fc3e06',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_09fc3e06',
    iconColumnHeader: 'iconColumnHeader_09fc3e06',
    fileRow: 'fileRow_09fc3e06',
    folderRow: 'folderRow_09fc3e06',
    emptyFolder: 'emptyFolder_09fc3e06',
    emptyFolderImage: 'emptyFolderImage_09fc3e06',
    emptyFolderImageTag: 'emptyFolderImageTag_09fc3e06',
    emptyFolderTitle: 'emptyFolderTitle_09fc3e06',
    emptyFolderSubText: 'emptyFolderSubText_09fc3e06',
    emptyFolderMobile: 'emptyFolderMobile_09fc3e06',
    emptyFolderPc: 'emptyFolderPc_09fc3e06',
    emptyFolderIcon: 'emptyFolderIcon_09fc3e06',
};
exports.default = styles;
/* tslint:enable */
