// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/SiteFilePickerTab/SiteFilePickerTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./SiteFilePickerTab.module.css');
var styles = {
    filePicker: 'filePicker_bc2e9b77',
    tabsContainer: 'tabsContainer_bc2e9b77',
    tabContainer: 'tabContainer_bc2e9b77',
    nav: 'nav_bc2e9b77',
    tabHeaderContainer: 'tabHeaderContainer_bc2e9b77',
    breadcrumbNav: 'breadcrumbNav_bc2e9b77',
    breadcrumbNavItem: 'breadcrumbNavItem_bc2e9b77',
    tab: 'tab_bc2e9b77',
    tabFiles: 'tabFiles_bc2e9b77',
    scrollablePaneWrapper: 'scrollablePaneWrapper_bc2e9b77',
    actionButtonsContainer: 'actionButtonsContainer_bc2e9b77',
    actionButtons: 'actionButtons_bc2e9b77',
    actionButton: 'actionButton_bc2e9b77',
    tabHeader: 'tabHeader_bc2e9b77',
    selectedTab: 'selectedTab_bc2e9b77',
    header: 'header_bc2e9b77',
    xlg: 'xlg_bc2e9b77',
    xxlg: 'xxlg_bc2e9b77',
    xxxlg: 'xxxlg_bc2e9b77',
    xxxxlg: 'xxxxlg_bc2e9b77',
    itemPickerTopBar: 'itemPickerTopBar_bc2e9b77',
    commandBarNoChevron: 'commandBarNoChevron_bc2e9b77',
    fileItem: 'fileItem_bc2e9b77',
    folderItem: 'folderItem_bc2e9b77',
    fileTypeIcon: 'fileTypeIcon_bc2e9b77',
    fileTypeIconIcon: 'fileTypeIconIcon_bc2e9b77',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_bc2e9b77',
    iconColumnHeader: 'iconColumnHeader_bc2e9b77',
    fileRow: 'fileRow_bc2e9b77',
    folderRow: 'folderRow_bc2e9b77',
    emptyFolder: 'emptyFolder_bc2e9b77',
    emptyFolderImage: 'emptyFolderImage_bc2e9b77',
    emptyFolderImageTag: 'emptyFolderImageTag_bc2e9b77',
    emptyFolderTitle: 'emptyFolderTitle_bc2e9b77',
    emptyFolderSubText: 'emptyFolderSubText_bc2e9b77',
    emptyFolderMobile: 'emptyFolderMobile_bc2e9b77',
    emptyFolderPc: 'emptyFolderPc_bc2e9b77',
    emptyFolderIcon: 'emptyFolderIcon_bc2e9b77',
};
exports.default = styles;
/* tslint:enable */
