// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/FilePicker.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FilePicker.module.css');
var styles = {
    filePicker: 'filePicker_6a88afee',
    tabsContainer: 'tabsContainer_6a88afee',
    tabContainer: 'tabContainer_6a88afee',
    nav: 'nav_6a88afee',
    tabHeaderContainer: 'tabHeaderContainer_6a88afee',
    breadcrumbNav: 'breadcrumbNav_6a88afee',
    breadcrumbNavItem: 'breadcrumbNavItem_6a88afee',
    tab: 'tab_6a88afee',
    tabFiles: 'tabFiles_6a88afee',
    scrollablePaneWrapper: 'scrollablePaneWrapper_6a88afee',
    actionButtonsContainer: 'actionButtonsContainer_6a88afee',
    actionButtons: 'actionButtons_6a88afee',
    actionButton: 'actionButton_6a88afee',
    tabHeader: 'tabHeader_6a88afee',
    selectedTab: 'selectedTab_6a88afee',
    header: 'header_6a88afee',
    xlg: 'xlg_6a88afee',
    xxlg: 'xxlg_6a88afee',
    xxxlg: 'xxxlg_6a88afee',
    xxxxlg: 'xxxxlg_6a88afee',
    itemPickerTopBar: 'itemPickerTopBar_6a88afee',
    commandBarNoChevron: 'commandBarNoChevron_6a88afee',
    fileItem: 'fileItem_6a88afee',
    folderItem: 'folderItem_6a88afee',
    fileTypeIcon: 'fileTypeIcon_6a88afee',
    fileTypeIconIcon: 'fileTypeIconIcon_6a88afee',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_6a88afee',
    iconColumnHeader: 'iconColumnHeader_6a88afee',
    fileRow: 'fileRow_6a88afee',
    folderRow: 'folderRow_6a88afee',
    emptyFolder: 'emptyFolder_6a88afee',
    emptyFolderImage: 'emptyFolderImage_6a88afee',
    emptyFolderImageTag: 'emptyFolderImageTag_6a88afee',
    emptyFolderTitle: 'emptyFolderTitle_6a88afee',
    emptyFolderSubText: 'emptyFolderSubText_6a88afee',
    emptyFolderMobile: 'emptyFolderMobile_6a88afee',
    emptyFolderPc: 'emptyFolderPc_6a88afee',
    emptyFolderIcon: 'emptyFolderIcon_6a88afee',
};
exports.default = styles;
/* tslint:enable */
