// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/RecentFilesTab/RecentFilesTab.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./RecentFilesTab.module.css');
var styles = {
    filePicker: 'filePicker_88724665',
    tabsContainer: 'tabsContainer_88724665',
    tabContainer: 'tabContainer_88724665',
    nav: 'nav_88724665',
    tabHeaderContainer: 'tabHeaderContainer_88724665',
    breadcrumbNav: 'breadcrumbNav_88724665',
    breadcrumbNavItem: 'breadcrumbNavItem_88724665',
    tab: 'tab_88724665',
    tabFiles: 'tabFiles_88724665',
    scrollablePaneWrapper: 'scrollablePaneWrapper_88724665',
    actionButtonsContainer: 'actionButtonsContainer_88724665',
    actionButtons: 'actionButtons_88724665',
    actionButton: 'actionButton_88724665',
    tabHeader: 'tabHeader_88724665',
    selectedTab: 'selectedTab_88724665',
    header: 'header_88724665',
    xlg: 'xlg_88724665',
    xxlg: 'xxlg_88724665',
    xxxlg: 'xxxlg_88724665',
    xxxxlg: 'xxxxlg_88724665',
    itemPickerTopBar: 'itemPickerTopBar_88724665',
    commandBarNoChevron: 'commandBarNoChevron_88724665',
    fileItem: 'fileItem_88724665',
    folderItem: 'folderItem_88724665',
    fileTypeIcon: 'fileTypeIcon_88724665',
    fileTypeIconIcon: 'fileTypeIconIcon_88724665',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_88724665',
    iconColumnHeader: 'iconColumnHeader_88724665',
    fileRow: 'fileRow_88724665',
    folderRow: 'folderRow_88724665',
    emptyFolder: 'emptyFolder_88724665',
    emptyFolderImage: 'emptyFolderImage_88724665',
    emptyFolderImageTag: 'emptyFolderImageTag_88724665',
    emptyFolderTitle: 'emptyFolderTitle_88724665',
    emptyFolderSubText: 'emptyFolderSubText_88724665',
    emptyFolderMobile: 'emptyFolderMobile_88724665',
    emptyFolderPc: 'emptyFolderPc_88724665',
    emptyFolderIcon: 'emptyFolderIcon_88724665',
    gridListCell: 'gridListCell_88724665',
    cellRight: 'cellRight_88724665',
    cellLastRow: 'cellLastRow_88724665',
    itemTile: 'itemTile_88724665',
    isFile: 'isFile_88724665',
    isPhoto: 'isPhoto_88724665',
    isVideo: 'isVideo_88724665',
    hasThumbnail: 'hasThumbnail_88724665',
    itemTileContent: 'itemTileContent_88724665',
    itemTileDiscoverFileContainer: 'itemTileDiscoverFileContainer_88724665',
    itemTileFileContainer: 'itemTileFileContainer_88724665',
    itemTileFile: 'itemTileFile_88724665',
    itemTileThumbnail: 'itemTileThumbnail_88724665',
    image: 'image_88724665',
    itemTileCheckCircle: 'itemTileCheckCircle_88724665',
    isSelected: 'isSelected_88724665',
    itemTileNamePlate: 'itemTileNamePlate_88724665',
    itemTileName: 'itemTileName_88724665',
    itemTileChildCount: 'itemTileChildCount_88724665',
    itemTileSubText: 'itemTileSubText_88724665',
    recentGridList: 'recentGridList_88724665',
};
exports.default = styles;
/* tslint:enable */
