// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/controls/DocumentTile/DocumentTile.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./DocumentTile.module.css');
var styles = {
    filePicker: 'filePicker_9921d9b7',
    tabsContainer: 'tabsContainer_9921d9b7',
    tabContainer: 'tabContainer_9921d9b7',
    nav: 'nav_9921d9b7',
    tabHeaderContainer: 'tabHeaderContainer_9921d9b7',
    breadcrumbNav: 'breadcrumbNav_9921d9b7',
    breadcrumbNavItem: 'breadcrumbNavItem_9921d9b7',
    tab: 'tab_9921d9b7',
    tabFiles: 'tabFiles_9921d9b7',
    scrollablePaneWrapper: 'scrollablePaneWrapper_9921d9b7',
    actionButtonsContainer: 'actionButtonsContainer_9921d9b7',
    actionButtons: 'actionButtons_9921d9b7',
    actionButton: 'actionButton_9921d9b7',
    tabHeader: 'tabHeader_9921d9b7',
    selectedTab: 'selectedTab_9921d9b7',
    header: 'header_9921d9b7',
    xlg: 'xlg_9921d9b7',
    xxlg: 'xxlg_9921d9b7',
    xxxlg: 'xxxlg_9921d9b7',
    xxxxlg: 'xxxxlg_9921d9b7',
    itemPickerTopBar: 'itemPickerTopBar_9921d9b7',
    commandBarNoChevron: 'commandBarNoChevron_9921d9b7',
    fileItem: 'fileItem_9921d9b7',
    folderItem: 'folderItem_9921d9b7',
    fileTypeIcon: 'fileTypeIcon_9921d9b7',
    fileTypeIconIcon: 'fileTypeIconIcon_9921d9b7',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_9921d9b7',
    iconColumnHeader: 'iconColumnHeader_9921d9b7',
    fileRow: 'fileRow_9921d9b7',
    folderRow: 'folderRow_9921d9b7',
    emptyFolder: 'emptyFolder_9921d9b7',
    emptyFolderImage: 'emptyFolderImage_9921d9b7',
    emptyFolderImageTag: 'emptyFolderImageTag_9921d9b7',
    emptyFolderTitle: 'emptyFolderTitle_9921d9b7',
    emptyFolderSubText: 'emptyFolderSubText_9921d9b7',
    emptyFolderMobile: 'emptyFolderMobile_9921d9b7',
    emptyFolderPc: 'emptyFolderPc_9921d9b7',
    emptyFolderIcon: 'emptyFolderIcon_9921d9b7',
    aboveNameplate: 'aboveNameplate_9921d9b7',
    tileContent: 'tileContent_9921d9b7',
    foreground: 'foreground_9921d9b7',
    grid: 'grid_9921d9b7',
    cellContent: 'cellContent_9921d9b7',
    content: 'content_9921d9b7',
    tile: 'tile_9921d9b7',
    invokable: 'invokable_9921d9b7',
    namePlate: 'namePlate_9921d9b7',
    nameplate: 'nameplate_9921d9b7',
    name: 'name_9921d9b7',
    isLarge: 'isLarge_9921d9b7',
    activity: 'activity_9921d9b7',
    signalField: 'signalField_9921d9b7',
    compact: 'compact_9921d9b7',
    signalFieldValue: 'signalFieldValue_9921d9b7',
    folderList: 'folderList_9921d9b7',
    link: 'link_9921d9b7',
    cell: 'cell_9921d9b7',
    listCell: 'listCell_9921d9b7',
    label: 'label_9921d9b7',
    check: 'check_9921d9b7',
    selectable: 'selectable_9921d9b7',
    showCheck: 'showCheck_9921d9b7',
    selected: 'selected_9921d9b7',
    dialogMainOverride: 'dialogMainOverride_9921d9b7',
    itemTileBlankCover: 'itemTileBlankCover_9921d9b7',
    metadata: 'metadata_9921d9b7',
    isSmall: 'isSmall_9921d9b7',
    listPage: 'listPage_9921d9b7',
    odItemTile2FolderCover: 'odItemTile2FolderCover_9921d9b7',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_9921d9b7',
    odItemTile2Image: 'odItemTile2Image_9921d9b7',
    odImageFrame2: 'odImageFrame2_9921d9b7',
    odImageFrame2Image: 'odImageFrame2Image_9921d9b7',
    odImageFrame: 'odImageFrame_9921d9b7',
    odImageStack: 'odImageStack_9921d9b7',
    odImageStackTile: 'odImageStackTile_9921d9b7',
    odImageTile: 'odImageTile_9921d9b7',
    odImageTileBackground: 'odImageTileBackground_9921d9b7',
    odImageTileImage: 'odImageTileImage_9921d9b7',
    odItemTile2SmallIcon: 'odItemTile2SmallIcon_9921d9b7',
};
exports.default = styles;
/* tslint:enable */
