// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/controls/FileBrowser/FileBrowser.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./FileBrowser.module.css');
var styles = {
    filePicker: 'filePicker_788cea08',
    tabsContainer: 'tabsContainer_788cea08',
    tabContainer: 'tabContainer_788cea08',
    nav: 'nav_788cea08',
    tabHeaderContainer: 'tabHeaderContainer_788cea08',
    breadcrumbNav: 'breadcrumbNav_788cea08',
    breadcrumbNavItem: 'breadcrumbNavItem_788cea08',
    tab: 'tab_788cea08',
    tabFiles: 'tabFiles_788cea08',
    scrollablePaneWrapper: 'scrollablePaneWrapper_788cea08',
    actionButtonsContainer: 'actionButtonsContainer_788cea08',
    actionButtons: 'actionButtons_788cea08',
    actionButton: 'actionButton_788cea08',
    tabHeader: 'tabHeader_788cea08',
    selectedTab: 'selectedTab_788cea08',
    header: 'header_788cea08',
    xlg: 'xlg_788cea08',
    xxlg: 'xxlg_788cea08',
    xxxlg: 'xxxlg_788cea08',
    xxxxlg: 'xxxxlg_788cea08',
    itemPickerTopBar: 'itemPickerTopBar_788cea08',
    commandBarNoChevron: 'commandBarNoChevron_788cea08',
    fileItem: 'fileItem_788cea08',
    folderItem: 'folderItem_788cea08',
    fileTypeIcon: 'fileTypeIcon_788cea08',
    fileTypeIconIcon: 'fileTypeIconIcon_788cea08',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_788cea08',
    iconColumnHeader: 'iconColumnHeader_788cea08',
    fileRow: 'fileRow_788cea08',
    folderRow: 'folderRow_788cea08',
    emptyFolder: 'emptyFolder_788cea08',
    emptyFolderImage: 'emptyFolderImage_788cea08',
    emptyFolderImageTag: 'emptyFolderImageTag_788cea08',
    emptyFolderTitle: 'emptyFolderTitle_788cea08',
    emptyFolderSubText: 'emptyFolderSubText_788cea08',
    emptyFolderMobile: 'emptyFolderMobile_788cea08',
    emptyFolderPc: 'emptyFolderPc_788cea08',
    emptyFolderIcon: 'emptyFolderIcon_788cea08',
};
exports.default = styles;
/* tslint:enable */
