// @pnp/spfx-property-controls/lib/propertyFields/filePicker/filePickerControls/controls/TilesList/TilesList.module.scss.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
/* tslint:disable */
require('./TilesList.module.css');
var styles = {
    filePicker: 'filePicker_304e9a86',
    tabsContainer: 'tabsContainer_304e9a86',
    tabContainer: 'tabContainer_304e9a86',
    nav: 'nav_304e9a86',
    tabHeaderContainer: 'tabHeaderContainer_304e9a86',
    breadcrumbNav: 'breadcrumbNav_304e9a86',
    breadcrumbNavItem: 'breadcrumbNavItem_304e9a86',
    tab: 'tab_304e9a86',
    tabFiles: 'tabFiles_304e9a86',
    scrollablePaneWrapper: 'scrollablePaneWrapper_304e9a86',
    actionButtonsContainer: 'actionButtonsContainer_304e9a86',
    actionButtons: 'actionButtons_304e9a86',
    actionButton: 'actionButton_304e9a86',
    tabHeader: 'tabHeader_304e9a86',
    selectedTab: 'selectedTab_304e9a86',
    header: 'header_304e9a86',
    xlg: 'xlg_304e9a86',
    xxlg: 'xxlg_304e9a86',
    xxxlg: 'xxxlg_304e9a86',
    xxxxlg: 'xxxxlg_304e9a86',
    itemPickerTopBar: 'itemPickerTopBar_304e9a86',
    commandBarNoChevron: 'commandBarNoChevron_304e9a86',
    fileItem: 'fileItem_304e9a86',
    folderItem: 'folderItem_304e9a86',
    fileTypeIcon: 'fileTypeIcon_304e9a86',
    fileTypeIconIcon: 'fileTypeIconIcon_304e9a86',
    detailsListHeaderFileTypeIcon: 'detailsListHeaderFileTypeIcon_304e9a86',
    iconColumnHeader: 'iconColumnHeader_304e9a86',
    fileRow: 'fileRow_304e9a86',
    folderRow: 'folderRow_304e9a86',
    emptyFolder: 'emptyFolder_304e9a86',
    emptyFolderImage: 'emptyFolderImage_304e9a86',
    emptyFolderImageTag: 'emptyFolderImageTag_304e9a86',
    emptyFolderTitle: 'emptyFolderTitle_304e9a86',
    emptyFolderSubText: 'emptyFolderSubText_304e9a86',
    emptyFolderMobile: 'emptyFolderMobile_304e9a86',
    emptyFolderPc: 'emptyFolderPc_304e9a86',
    emptyFolderIcon: 'emptyFolderIcon_304e9a86',
    aboveNameplate: 'aboveNameplate_304e9a86',
    tileContent: 'tileContent_304e9a86',
    foreground: 'foreground_304e9a86',
    grid: 'grid_304e9a86',
    cellContent: 'cellContent_304e9a86',
    content: 'content_304e9a86',
    tile: 'tile_304e9a86',
    invokable: 'invokable_304e9a86',
    namePlate: 'namePlate_304e9a86',
    nameplate: 'nameplate_304e9a86',
    name: 'name_304e9a86',
    isLarge: 'isLarge_304e9a86',
    activity: 'activity_304e9a86',
    signalField: 'signalField_304e9a86',
    compact: 'compact_304e9a86',
    signalFieldValue: 'signalFieldValue_304e9a86',
    folderList: 'folderList_304e9a86',
    link: 'link_304e9a86',
    cell: 'cell_304e9a86',
    listCell: 'listCell_304e9a86',
    label: 'label_304e9a86',
    check: 'check_304e9a86',
    selectable: 'selectable_304e9a86',
    showCheck: 'showCheck_304e9a86',
    selected: 'selected_304e9a86',
    dialogMainOverride: 'dialogMainOverride_304e9a86',
    itemTileBlankCover: 'itemTileBlankCover_304e9a86',
    metadata: 'metadata_304e9a86',
    isSmall: 'isSmall_304e9a86',
    listPage: 'listPage_304e9a86',
    odItemTile2FolderCover: 'odItemTile2FolderCover_304e9a86',
    standaloneListBreadcrumb: 'standaloneListBreadcrumb_304e9a86',
};
exports.default = styles;
/* tslint:enable */
